website/styles/globals.css

```diff
@@ -0,0 +1,55 @@
+html {
```

Comment:
What's the reason for globals being in a dedicated `.css` file rather than using Emotion's global style declarations?